scripts/create-merged-schema.ts

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env ts-node
+/*
+ * Create a complete JSON schema for tsconfig.json
+ * by merging the schemastore schema with our ts-node additions.
+ * This merged schema can be submitted in a pull request to
+ * SchemaStore.
+ */
```

Comment:
Is this script necessary? Why can't we specify a URL within our schema to have it already be up-to-date with the remote schema? I'm worried this will mean that every TypeScript version I'll need to be generating and releasing the updated schema. You're meant to be able to use `$ref` for this use-case: https://cswr.github.io/JsonSchema/spec/definitions_references/.

Reply:
Correct, I'm generating 2 schemas.  First is `tsconfig.schema.json`, which *does* use `$ref`.  That was my initial approach, same thinking as you.

@G-Rath suggested patching ts-node's additions into the official file on Schemastore.  This script produces that merged schema.  The idea is that it would live in SchemaStore, so editors would tab-complete the ts-node fields without any configuration on the user's part.  I'm not sure if they'll ever merge it, though, because if someone tab-completes the ts-node options, they'll still need to install ts-node.  It could be nice publicity for the library; I dunno.

Reply:
They will merge it, since it's not a breaking change :)

The configuration options are harmless if `ts-node` isn't there.